.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Installation artifacts
 src
+requirements/*.txt
```

Comment:
Are these generated by something else? It seems like we don't want to ignore these?